src/CsvReader.php

```diff
@@ -36,29 +36,43 @@ class CsvReader extends AbstractCsvFile implements \Iterator
      */
     protected $header;
 
+    /**
+     * @var string
+     */
+    private $fileName;
```

Comment:
Nechceš si nastavit oneline field docy? 
![image](https://user-images.githubusercontent.com/642928/39521146-946339fc-4e0d-11e8-9fef-7a289ed2ae3c.png)

Reply:
ja ti nevim, me se to nejak nelibi :) premyslel jsem nad tim proc a asi mam dva duvody, jednak k fieldum nekdy pisu komentar k cemu jsou a jednak me drazdi kdyz je phpdoc u metody jinej nez u fieldu

Reply:
Motivace proč to používat je veskrze:
* fieldy jsou na začátku třídy a takhle máš 1(blank)+3(doc)+1(def) x N místo 1+1+1 x N řádků. Tj. prostě víc scrollovaní
* fieldy většinou (!) nemívají description a ostatní součásti docblocku (a pokud ano, tak ho můžeš udělat víceřádkový

Ale více méně souhlasím, že i tvoje argumenty jsou validní ;)